packages/compass-components/src/components/guide-cue/guide-cue.tsx

```diff
@@ -45,8 +82,8 @@ export type GuideCueProps<T> = Omit<
   'currentStep' | 'refEl' | 'numberOfSteps' | 'open' | 'setOpen' | 'children'
 > &
   GroupAndStep & {
-    description: string | React.ReactElement;
     cueId: string;
+    description: string | React.ReactElement;
```

Comment:
Feel free to ignore, but this (and in a few other places) is basically becoming a ReactChild type

```suggestion
    description: React.ReactChild;
```